RecoParticleFlow/PFClusterProducer/python/particleFlowRecHitECAL_cfi.py

```diff
@@ -15,10 +16,11 @@
            cms.PSet(
              name = cms.string("PFEBRecHitCreator"),
              src  = cms.InputTag("ecalRecHit","EcalRecHitsEB"),
+             srFlags = cms.InputTag("ecalDigis"),
              qualityTests = cms.VPSet(
                   cms.PSet(
-                  name = cms.string("PFRecHitQTestThreshold"),
-                  threshold = cms.double(0.08)
+                  name = cms.string("PFRecHitQTestECALThreshold"),
+                  thresholds = particle_flow_zero_suppression_ECAL.thresholds
```

Comment:
is the effect of PFRecHitQTestECALThreshold with these thresholds  a strict subset of PFRecHitQTestThreshold with 0.08 threshold?
If so, it may be easier to add PFRecHitQTestECALThreshold as an extra QTest in the era (rather than searching in the qualityTests VPSet for the right value to edit)